# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/JitFrames.h
# Commit: 364d4e24bc13
# Full Hash: 364d4e24bc13eafbb893adf6381a6cbb862d405f
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2024-07-29 20:43:57
# Regressor Bug: 1909224
# File Overlap Count: 2
# Description:
#   Bug 1909224 part 3 - Remove checks for FailInstanceReg on JIT => Wasm call path. r=rhunt
#   
#   Because we now unwind JS JIT frames after unwinding Wasm frames, we no longer
#   return to the JIT => Wasm entry code from the exception handler.
#   
# ==============================================================================

diff -r 6eb32b942a04 -r 364d4e24bc13 js/src/jit/JitFrames.h
--- a/js/src/jit/JitFrames.h	Mon Jul 29 13:14:47 2024 +0000
+++ b/js/src/jit/JitFrames.h	Mon Jul 29 13:14:48 2024 +0000
@@ -113,9 +113,8 @@
   // Jump to the bailout tail stub.
   Bailout,
 
-  // The innermost frame was a wasm frame.
-  // Return to the wasm entry frame.
-  Wasm,
+  // Return to the wasm interpreter entry frame.
+  WasmInterpEntry,
 
   // The exception was caught by a wasm catch handler.
   // Restore state and jump to it.